src/python/py/models/README.md

```diff
@@ -4,24 +4,27 @@ This folder contains the model builder for quickly creating optimized and quanti
 
 # Contents
 
-- [Current Support](#current-support)
-- [Usage](#usage)
-  - [Full Usage](#full-usage)
-  - [Original PyTorch Model from Hugging Face](#original-pytorch-model-from-hugging-face)
-  - [Original PyTorch Model from Disk](#original-pytorch-model-from-disk)
-  - [Customized or Finetuned PyTorch Model](#customized-or-finetuned-pytorch-model)
-  - [Quantized PyTorch Model](#quantized-pytorch-model)
-  - [GGUF Model](#gguf-model)
-  - [Extra Options](#extra-options)
-    - [Config Only](#config-only)
-    - [Exclude Embedding Layer](#exclude-embedding-layer)
-    - [Exclude Language Modeling Head](#exclude-language-modeling-head)
-    - [Enable Cuda Graph](#enable-cuda-graph)
-    - [Use 8 Bits Quantization in QMoE](#use-8-bits-quantization-in-qmoe)
```

Comment:
Can we just insert the new "Hugging Face Authentication" section here and then keep the table of contents formatting as is? I think the first two links are self-explanatory since a user can see the README title and the table of contents heading as soon as the README doc is opened.

My concern is that users may click on those links, not see the page move since the user is already at those positions in the README, and then wonder why the link is not working (even though it is).